src/include/sof/lib/memory.h

```diff
@@ -10,4 +10,12 @@
 
 #include <platform/lib/memory.h>
 
+#ifdef CONFIG_IMX8ULP
+#define host_to_local(addr) (SDRAM0_BASE + ((addr) - MEM_RESERVED))
```

Comment:
This conversion will be platform memory map specific so we will need it defined in platform/include/lib/memory.h.
This will be a NOP for all platforms except IMX8ULP

Reply:
@lgirdwood do you mean we should add this definition in every src/platform/<$platform/include/platform/lib/memory.h file right?
And make it NOP for all other places except imx8ulp?

Reply:
Correctted